README.md

```diff
@@ -48,6 +49,8 @@ To get your credentials:
 For Connect, you can configure a default fees amount (`fees_amount`) or percentage (`fees_percent`, such as .3 for 30%). These can be modified on a per request basis by passing the plugin property `fees_amount` or `fees_percent`.
 You'll also need to add a row to the `stripe_application_fees` table and add a percent (such as .3 for 30%) to the `application_fee` field.
 
+For dynamic descriptors, you can configure a dynamic descriptor template (`dynamic_descriptor`) with Ruby's string interpolation syntax. For example `"KILLBILL %{kb_payment_transaction_id}"`. Available variables are `kb_account_id`, `kb_payment_id`, `kb_payment_transaction_id`, `kb_payment_method_id`. Stripe limits to 22 characters for the charge descriptor so the first 22 characters of the description will be used.
```

Comment:
It's a good idea but are any of these options usable in practice? A UUID will already be 37 characters, way above the 22. What are you going to use personally?